.ci/integration-tests

```diff
@@ -224,6 +261,37 @@ function hf_check_ms_freeze() {
     fi
 }
 
+function hf_orphan_resources() {
+    if [ $1 == "aws" ]; then
+        # Check if there are any VMs matching the tag exists.
```

Comment:
Does it make sense to check for orphan NICs also here?

Reply:
As discussed skipping the nic part.